manic/checkout.py

```diff
@@ -280,8 +282,10 @@ def main(args):
                 printlog(msg)
             # exit gracefully
             msg = textwrap.fill(
-                'Some external repositories that are not in a clean '
-                'state. Please ensure all external repositories are clean '
+                'Some external repositories are not in a clean state. '
+                '(Generally, these are repositories with "M" in the '
```

Comment:
@bandre-ucar can you check this statement in parentheses? If it's *always* the case that problematic repos are ones with 'M' in the second column, then we could remove the "Generally"

Reply:
"Generally" isn't necessary. They are always the repos with M in the second column. But this message will be changed with GH-51, so don't bother to update it.